ci.conveyor.conf

```diff
@@ -14,6 +14,17 @@ app {
       private-key = ${env.APPLE_NOTARIZATION_KEY_PATH}
     }
   }
+  windows {
+      store {
+          identity-name = "ApadmiLtd.Mockzilla"
+          publisher = "CN=20FB62E1-0B01-4C2B-B6F3-91918995E8A9"
```

Comment:
I'm guessing these values aren't sensitive?

Reply:
Nah they end up in the manifest that is part of the download